pkg/api/validation/validation_test.go

```diff
@@ -2792,10 +2792,14 @@ func TestAlphaHugePagesIsolation(t *testing.T) {
 						Name: "ctr", Image: "image", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: "File",
 						Resources: api.ResourceRequirements{
 							Requests: api.ResourceList{
-								api.ResourceName("hugepages-2Mi"): resource.MustParse("1Gi"),
+								api.ResourceName(api.ResourceCPU):    resource.MustParse("10"),
+								api.ResourceName(api.ResourceMemory): resource.MustParse("10G"),
+								api.ResourceName("hugepages-2Mi"):    resource.MustParse("1Gi"),
 							},
 							Limits: api.ResourceList{
-								api.ResourceName("hugepages-2Mi"): resource.MustParse("1Gi"),
+								api.ResourceName(api.ResourceCPU):    resource.MustParse("10"),
+								api.ResourceName(api.ResourceMemory): resource.MustParse("10G"),
+								api.ResourceName("hugepages-2Mi"):    resource.MustParse("1Gi"),
```

Comment:
would you please add some tests about ignoring the cpu or memory and raise the errors?